### PR TITLE
Operationbranch

### PR DESCRIPTION
**Pull request by jyothiperi**
Can your review mycode

### CODE REVIEW

File: operations.js [hunk 1]
```diff
@@ -2,7 +2,7 @@
 function performoperation(x, y, action) {    if (action === "+") {
       console.log(x + y)
```
Comment (Sravani-34) [inline operations.js]: Please add semi colon

File: operations.js [hunk 2]
```diff
@@ -2,7 +2,7 @@
...
     } else if (action === "-") {
-      console.log(x - y);
+      console.log(x - y)
```
Comment (Sravani-34) [inline operations.js]: Add semicolon

**Review by Sravani-34 — COMMENTED** [review]
Review done

**Review by Sravani-34 — APPROVED** [review]
Code is fine

**Comment by jyothiperi** [comment]
merge the file

**Comment by jyothiperi** [comment]
merge the file

**Comment by nidhisingh87** [comment]
Code is working

**Pull request closed without merging**